.java-version

```diff
@@ -1 +1 @@
-1.8
+11
```

Comment:
🎉

Reply:
This just affects jenv for local gradle tasks, but without this I was getting errors running locally